src/use/uWebSockets.ts

```diff
@@ -10,12 +11,16 @@ export interface Extra {
   /**
    * The actual socket connection between the server and the client.
    */
-  readonly socket: uWS.WebSocket;
+  readonly socket: uWS.WebSocket & { upgradeReq: Request };
   /**
    * The initial HTTP request before the actual
    * socket and connection is established.
    */
-  readonly request: uWS.HttpRequest;
+  readonly request: Request;
+}
+
+export interface Request {
+  headers: http.IncomingHttpHeaders;
```

Comment:
If you remove the `upgradeReq` from the `socket`, this can extended inline.
```suggestion
  readonly request: Request & { headers: http.IncomingHttpHeaders };
}
```

Reply:
Request is our own Interface with only one field: `headers`

Reply:
Ok. What I need to do? In your suggested change you deleted my `Request` interface but still used as type of `request` field